### PR TITLE
framework/ble_manager: fix BLE minor bug

### PR DESCRIPTION
**Pull request by an4967**
- fix BLE minor bug
- fix BLE build warnings

### CODE REVIEW

File: framework/src/ble_manager/ble_manager_state.c [hunk 1]
```diff
@@ -910,6 +910,12 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 			}
 		}
 
+		if (ctx == NULL) {
```
Comment (jeongchanKim) [inline framework/src/ble_manager/ble_manager_state.c:913]: If `ctx` is NULL, is it needed to run from 890 to 911?
If not, it is better to move this NULL checking at line 890.

Comment (an4967) [reply framework/src/ble_manager/ble_manager_state.c:913]: @jeongchanKim 
At line:907, ctx is set after matching BLE context table.
This line means a context is not found in the full table.

File: framework/src/ble_manager/ble_manager_state.c [hunk 2]
```diff
@@ -466,7 +466,7 @@ ble_result_e blemgr_handle_request(blemgr_msg_s *msg)
 		2. auto connection
 		*/
 		ret = TRBLE_SUCCESS;
-		ble_client_ctx *ctx = (ble_client_ctx *)msg->param;
+		ble_client_ctx_internal *ctx = (ble_client_ctx_internal *)msg->param;
```
Comment (Taejun-Kwon) [inline framework/src/ble_manager/ble_manager_state.c:469]: you should check validation(null check)of msg & param right before switch-case, not here

Comment (an4967) [reply framework/src/ble_manager/ble_manager_state.c:469]: 'msg' is guaranteed that it is not NULL.
It is always created in ble_msg_handler.

**Review by pillip8282 — APPROVED** [review]
LGTM

**Pull request merged**